rbi/core/array.rbi

```diff
@@ -2,323 +2,237 @@
 
 # Arrays are ordered, integer-indexed collections of any object.
 #
-# [Array](Array) indexing starts at 0, as in C or
-# Java. A negative index is assumed to be relative to the end of the
-# array---that is, an index of -1 indicates the last element of the array,
-# -2 is the next to last element in the array, and so on.
-#
-#
-# A new array can be created by using the literal constructor `[]` .
-# Arrays can contain different types of objects. For example, the array
-# below contains an
-# [Integer](https://ruby-doc.org/core-2.6.3/Integer.html), a
```

Comment:
It seems that the script deletes some useful information that was present before, notably links and syntax highlighting for code snippets that are embedded in the docs.